naiveproxy/Makefile

```
@@ -5,12 +5,12 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=naiveproxy
-PKG_VERSION:=91.0.4472.77-1
-PKG_RELEASE:=4
+PKG_VERSION:=92.0.4515.107-1
+PKG_RELEASE:=$(AUTORELEASE)
```

```suggestion
PKG_RELEASE:=1
```
Don't use AUTORELEASE, for the stable version.